packages/augur-sdk/src/state/index.ts

```diff
@@ -23,13 +23,23 @@ console.log("Starting web worker");
       augur.off(SubscriptionEventNames.CompleteSetsPurchased);
     });
 
+    augur.on(SubscriptionEventNames.Burn, (data: any): void => {
+      console.log(data);
+      augur.off(SubscriptionEventNames.Burn);
+    });
+
+    augur.on(SubscriptionEventNames.Approval, (data: any): void => {
+      console.log(data);
+      augur.off(SubscriptionEventNames.Approval);
+    });
+
     setTimeout(async () => {
```

Comment:
Why does this setTimeout(...) exist

Reply:
Removed it. It was there because of the web worker. The worker needs time to ingest data.